tools/whitelistworker.ts

```diff
@@ -0,0 +1,74 @@
+import {interacts} from '@openst/mosaic-contracts';
+import MosaicConfig from '@openst/mosaic-chains/lib/src/Config/MosaicConfig';
```

Comment:
This tool is in mosaic-chains and we are using **@openst/mosaic-chains** package 🤔 and package is not even in package.json. It doesn't compile.
Change the import statement to **import MosaicConfig from '../src/Config/MosaicConfig';**